batavia/types/Tuple.js

```diff
@@ -324,6 +324,12 @@ Tuple.prototype.__getitem__ = function(index) {
                 return this[idx]
             }
         }
+    } else if (types.isinstance(index, types.Bool)) {
+        if (index) {
+            return this[1]
```

Comment:
Ah, so, it looks like this is failing if the list is too short to support these indices. We'll have to throw IndexError if the list is too short or empty.